source/includes/atlas-search.rst

```diff
@@ -0,0 +1,7 @@
+.. note::
+
+   `Atlas Search <https://www.mongodb.com/cloud/atlas?tck=docs_server>`__
+   makes it easy to build fast, relevance-based search capabilities on top of
+   your MongoDB data. Try it today on
+   `MongoDB Atlas <https://www.mongodb.com/cloud/atlas?tck=docs_server>`__,
```

Comment:
I suspect the ``tck`` value here (used to track who sent a user to the atlas page) ought to be ``docs_node`` or ``docs_driver``. You may want to ask @jdestefano-mongo to double-check -- I'd guess that ``docs_server`` should only be used for links originating from the manual itself.